### PR TITLE
fix: mget with multiple keys in different slots but first two in the same slot will fail

### PR DESCRIPTION
**Pull request by LangInteger**
Motivation:

There are normal cases for `mget` command in clustered redis that will fail to execute with vert.x redis client

Conformance:

Your commits should be signed and you should have signed the Eclipse Contributor Agreement as explained in https://github.com/eclipse/vert.x/blob/master/CONTRIBUTING.md
Please also make sure you adhere to the code style guidelines: https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines

### CODE REVIEW

File: src/test/java/io/vertx/redis/client/test/RedisClusterTest.java [hunk 1]
```diff
@@ -929,6 +931,90 @@ public void mget(TestContext should) {
       });
   }
 
+  @Test(timeout = 30_000)
```
Comment (LangInteger) [inline src/test/java/io/vertx/redis/client/test/RedisClusterTest.java:934]: This test case cannot pass with original code

**Comment by LangInteger** [comment]
@vietj @pmlopes Please have a look at this

**Comment by LangInteger** [comment]
The newly merged code `Future.all` from commit [e8ae8274e](https://github.com/vert-x3/vertx-redis-client/commit/e8ae8274e31f836e6dacb51d9b9d44ac4bc2d581) cannot compile in my local. Could you please help me solve the problem @vietj

**Comment by vietj** [comment]
can you try refresh your snapshots @LangInteger ?

**Comment by vietj** [comment]
can you rebase your PR on the latest master commit of this project ?

**Comment by LangInteger** [comment]
> can you rebase your PR on the latest master commit of this project ?

@vietj  Thanks, I managed to solve the problem by rebasing onto the latest master commit and addressing the dependencies once more. Could you please review this Pull Request, or invite someone else to provide their input?

**Comment by LangInteger** [comment]
@vietj Can we get this merged?

**Comment by tsegismont** [comment]
@pmlopes any chance for you to review this one?

**Comment by LangInteger** [comment]
@pmlopes  echo echo

**Review by tsegismont — APPROVED** [review]
Thank you @LangInteger , looks good to me.

I'll take care of backporting

**Pull request merged**